recipe/meta.yaml

```diff
@@ -20,29 +21,40 @@ source:
     folder: constructor  # [win]
 
 build:
-  number: 0
+  number: 1
+  skip: true  # [py != 39]
   ignore_run_exports:
     - '*'
 
 requirements:
   build:
+    - python {{ python_version }}  # [build_platform != target_platform]
+    # These are not needed technically, but we need them here so the build env
+    # is solved as closely to host as possible
+    - pip  # [build_platform != target_platform]
+    - tqdm  # [build_platform != target_platform]
+    - pyinstaller  # [build_platform != target_platform]
+    - conda ={{ version }}  # [build_platform != target_platform]
+    - conda-package-handling >=1.6  # [build_platform != target_platform]
+  host:
     - pyinstaller
-    - python ={{ python_version }}
+    - python {{ python_version }}
+    - pip
+    - tqdm
     - conda ={{ version }}
     - conda-package-handling >=1.6
-    - tqdm
-    - menuinst >=1.4.18  # [win]
+    - menuinst ={{ menuinst_version }}  # [win]
   run_constrained:
```

Comment:
```suggestion
  run:
    - python
  run_constrained:
```

Reply:
Nah this feedstock is an exception. The linter is just not equipped to understand that this is a self-contained distribution. We can ignore.

The problem with this PR is that is extremely hacky and fragile 😬